packages/wonder-blocks-core/components/unique-id-provider.js

```diff
@@ -66,10 +66,9 @@ export default class UniqueIDProvider extends React.Component<Props> {
         const {children, mockOnFirstRender, scope} = this.props;
 
         // If this is our first render, we're going to stop right here.
+        // Note: `firstRender` will be `false` on the first render if this
```

Comment:
Should we just rename this variable? Suggestions:

- `initialSsrLevelRender`
- `placeholderRender`
- something similar

Reply:
Maybe `ssrRender`?